### PR TITLE
Add documentation on OpenTelemetry spans

### PR DESCRIPTION
**Pull request by jkwatson**

### CODE REVIEW

File: opentelemetry/OpenTelemetry-Spans.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# OpenTelemetry Span Exporters
+
+For OpenTelemetry exporters use a `"collector.name"` of `"newrelic-opentelemetry-exporter"`
```
Comment (willnewrelic) [inline opentelemetry/OpenTelemetry-Spans.md]: Would you mind putting this in the ordered list?

Comment (jkwatson) [reply opentelemetry/OpenTelemetry-Spans.md]: I can do that!

Comment (jkwatson) [reply opentelemetry/OpenTelemetry-Spans.md]: done!

File: Guidelines.md [hunk 2]
```diff
@@ -32,9 +32,10 @@ represents, you might call that attribute `histogram.bucket`.
 ### Common patterns:
 * All metric names SHOULD start with the customer-provided metric name that the metric library provides to the exporter.
 * All metrics that share a common name MUST also have at least one variable attribute, so they can be independently aggregated by the back end.
-* All generated metrics should include an attribute `instrumentation.provider` which identifies the source of the telemetry (for example, "dropwizard" or "prometheus")
-* All generated metrics should include (if applicable) an attribute `instrumentation.name` which identifies what is being instrumented. (for example, "spring-framework" or "python-twisted")
-* All generated metrics should include an attribute `collector.name`, which identifies the particular exporter in question (for example, "newrelic-dropwizard-reporter", or "newrelic-opentelemetry-exporter")
+* All generated metrics/spans should include an attribute `instrumentation.provider` which identifies the source of the telemetry (for example, "dropwizard" or "prometheus")
```
Comment (MrAlias) [inline Guidelines.md]:
```suggestion
* All generated metrics/spans SHOULD include an attribute `instrumentation.provider` which identifies the source of the telemetry (for example, "dropwizard" or "prometheus")
```

File: Guidelines.md [hunk 3]
```diff
@@ -32,9 +32,10 @@ represents, you might call that attribute `histogram.bucket`.
...
+* All generated metrics/spans should include (if applicable) an attribute `instrumentation.name` which identifies what is being instrumented. (for example, "spring-framework" or "python-twisted")
```
Comment (MrAlias) [inline Guidelines.md]:
```suggestion
* All generated metrics/spans SHOULD include (if applicable) an attribute `instrumentation.name` which identifies what is being instrumented. (for example, "spring-framework" or "python-twisted")
```

File: Guidelines.md [hunk 4]
```diff
@@ -32,9 +32,10 @@ represents, you might call that attribute `histogram.bucket`.
...
+* All generated metrics/spans should include (in applicable) an attribute `instrumentation.version` which identifies the version of the instrumentation that generated the telemetry.
```
Comment (MrAlias) [inline Guidelines.md]:
```suggestion
* All generated metrics/spans SHOULD include (in applicable) an attribute `instrumentation.version` which identifies the version of the instrumentation that generated the telemetry.
```

File: Guidelines.md [hunk 5]
```diff
@@ -32,9 +32,10 @@ represents, you might call that attribute `histogram.bucket`.
...
+* All generated metrics/spans should include an attribute `collector.name`, which identifies the particular exporter in question (for example, "newrelic-dropwizard-reporter", or "newrelic-opentelemetry-exporter")
```
Comment (MrAlias) [inline Guidelines.md]:
```suggestion
* All generated metrics/spans SHOULD include an attribute `collector.name`, which identifies the particular exporter in question (for example, "newrelic-dropwizard-reporter", or "newrelic-opentelemetry-exporter")
```

File: opentelemetry/OpenTelemetry-Spans.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+# OpenTelemetry Span Exporters
+
+For OpenTelemetry exporters .
```
Comment (MrAlias) [inline opentelemetry/OpenTelemetry-Spans.md]:
```suggestion
For OpenTelemetry exporters.
```

**Pull request merged**